ui/app/components/app/account-menu/index.scss

```diff
@@ -37,13 +37,13 @@
   }
 
   &__lock-button {
+    @include H7;
+
     border: 1px solid $dusty-gray;
     background-color: transparent;
     color: $white;
     border-radius: 4px;
-    font-size: 12px;
-    line-height: 23px;
-    padding: 0 24px;
+    padding: 4px 24px;
```

Comment:
Ah, I was off by one. 3px would be 1px too small. It looks like 3.5px padding is equivalent to the old line-height.

Definitely not a blocker, but it might be nice to stick to 3.5, as vertical space is very limited in this menu.

Reply:
@Gudahtt i changed to 3.5 as per your suggestion